inst/@sym/solve.m

```diff
@@ -101,11 +120,10 @@
             '    else:'
             '        symbols.append(arg)'
             '#'
-            'if len(symbols) > 0:'
-            '    d = sp.solve(eqs, symbols, dict=True)'
-            'else:'
-            '    d = sp.solve(eqs, dict=True)'
+            'd = sp.solve(eqs, *symbols, dict=True)'
             '#'
+            'if not isinstance(d, (list, tuple)):'
+            '    return d,'
```

Comment:
The specific reason of that line is this issue: https://github.com/sympy/sympy/issues/11661